tempo/update.sh

```diff
@@ -0,0 +1,117 @@
+# This script is used to update the Tempo alerts from mixins from the upstream repository.
+#
+# Usage:
+#  ./tempo/update.sh from the root of the repository
+
+set -eau o pipefail
+
+BRANCHREF="heads/main"
+MIXIN_URL="https://raw.githubusercontent.com/grafana/tempo/refs/$BRANCHREF/operations/tempo-mixin-compiled/alerts.yaml"
+OUTPUT_FILE="$(pwd)/helm/prometheus-rules/templates/platform/atlas/alerting-rules/tempo-mixins.rules.yml"
+
+print_header() {
+echo '---
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+  name: tempo-mixins.rules
+  namespace: {{ .Values.namespace  }}
+spec:
+  groups:
+    - name: tempo_alerts
+      rules:'
+}
+
+apply_global_sed_fixes() {
+    local rule="$*"
+    echo "$rule" | jq -r |
+      sed 's/cluster/cluster_id/g' | # Rename cluster to cluster_id
+      sed 's/cluster_id,/cluster_id, installation, pipeline, provider,/g' | # Add mandatory labels
+      sed 's/{\([a-z]\)/{cluster_type=\\\"management_cluster\\\", \1/g' | # Only apply this alert to management clusters - this one is where there's already a selector
```

Comment:
Nit: Shouldn't we catch everything betwen `{}` ?

```suggestion
      sed 's/{\(.*\)}/{cluster_type=\\\"management_cluster\\\", \1}/g' | # Only apply this alert to management clusters - this one is where there's already a selector
```

Reply:
Ok got it.

Reply:
oh it works when capturing only the curly braces groups individually

````
$ echo 'queries{status="ok"} / queries{status="failed}'|sed -e 's/{\([^}]\+\)}/{cluster_type=\"management_cluster\", \1}/g'
queries{cluster_type="management_cluster", status="ok"} / queries{cluster_type="management_cluster", status="failed}
````